contracts/MANACrowdsale.sol

```diff
@@ -79,7 +79,10 @@ contract MANACrowdsale is ContinuousCrowdsale, WhitelistedCrowdsale, CappedCrowd
     }
 
     // low level token purchase function
-    function processPurchase(address beneficiary) internal returns(uint256) {
+    function buyTokens(address beneficiary) payable {
```

Comment:
I agree. Comment was copied verbatim from the OpenZeppelin [Crowdsale](https://github.com/OpenZeppelin/zeppelin-solidity/blob/master/contracts/crowdsale/Crowdsale.sol#L68) contract.